doc/source/man.rst

```diff
@@ -32,206 +32,217 @@ rho performs four major tasks:
 
-Creating and Editing Authentication Credentials
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+When a scan runs, it uses a network profile that contains the hosts or IP addresses to be accessed. The network profile also contains references to the authentication profiles that are required to access those systems. A single network profile can contain a reference to multiple authentication profiles as needed to connect to all systems in that network.
 
-rho uses SSH credentials to access the servers to get their system information during discovery. These credentials can be either a username-password or username-key pair. Each set of credentials is stored in a separate entry.
+Creating and Editing Authentication Profiles
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-**rho auth add --name=** *name* **--username=** *username* **[--password]** **[--sshkeyfile=** *key_file* **]** **[--sudo-password]** **[--vault=** *vault_file* **]**
+To create an authentication profile, supply SSH credentials as either a username-password pair or a username-key pair. Rho stores each set of credentials in a separate authentication profile entry.
 
-``--name=name``
+**rho auth add --name=** *name* **--username=** *username* **(--password | --sshkeyfile=** *key_file* **)** **[--sudo-password]** **[--vault=** *vault_file* **]**
 
-  This required argument sets the name of the new authentication credentials entry. This should be descriptive, such as identifying the user or server it relates to. For example, ``"server1-rhouser"``. It should never contain the actual password, as this name may be logged or printed during rho execution.
+``--name=name``
 
+  Required. Sets the name of the new authentication profile. Use a descriptive name for the value that is meaningful to your organization. For example, you could identify the user or server that the authentication profile relates to, for example, ``admin12`` or ``server1_jdoe``. Do not include the password as part of the name value, because this name might be logged or printed during rho execution.
 
 ``--username=username``
 
-  This required argument contains the username of the SSH identity will use to bind to the server.
+  Required. Sets the username of the SSH identity that is used to bind to the server.
 
 ``--password``
 
-  The argument prompts for the password for the --username identity.
+  Prompts for the password for the ``--username`` identity. Mutually exclusive with the ``--sshkeyfile`` option.
 
 ``--sshkeyfile=key_file``
 
-  Optionally, this contains the path and filename of the file containing the SSH key issued for the --username identity.
+  Sets the path and file name of the file that contains the SSH key for the ``--username`` identity. Mutually exclusive with the ``--password`` option.
```

Comment:
Should we be explicit and say that is the *private* SSH key path Rho is looking for?

Reply:
If anyone could possibly be confused, then yes.

Change made: "that contains the private SSH key"